folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -113,6 +116,34 @@ public static CompletionStage<RawMemcacheClient> connect(
       final int maxSetLength,
       final EventLoopGroup eventLoopGroup,
       final Class<? extends Channel> channelClass) {
+    return connect(
+        address,
+        outstandingRequestLimit,
+        eventLoopThreadFlushMaxBatchSize,
+        binary,
+        executor,
+        connectionTimeoutMillis,
+        charset,
+        metrics,
+        maxSetLength,
+        eventLoopGroup,
+        channelClass,
+        null);
+  }
+
+  public static CompletionStage<RawMemcacheClient> connect(
```

Comment:
Does this need to be public?

Reply:
It is called from ReconnectingClient (in different package).